### PR TITLE
DAM crunch group collaboration

### PR DESCRIPTION
**Pull request by n8wood-brown**

### CODE REVIEW

**Comment by n8wood-brown** [comment]
DAM crunch group collaboration stub and an update to the collaboration schema file.

File: our-work/collaborations/crunch.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+title: The Crunch Group
+id: crunch-group
+description: The CRUNCH group is a research team of the Division of Applied Mathematics working at the interface of Computational Mathematics + Machine Learning + X, where X may be problems in biology, geophysics, soft matter, functional materials, physical chemistry, or fluid and solid mechanics. Their emphasis is on Scientific Machine Learning and Physics-Informed Neural Networks (PINNs) that the CRUNCH group pioneered.
```
Comment (mirestrepo) [inline our-work/collaborations/crunch.yml:3]: At the end of the description add a sentence or two about CCV engagement's with the group. For instance in other collars we have said
>CCV's collaboration with BrainGate focused around the development of a database solution using TimeScaleDB for continuous at-home neuro-recording.

File: our-work/collaborations/crunch.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+title: The Crunch Group
+id: crunch-group
+description: The CRUNCH group is a research team of the Division of Applied Mathematics working at the interface of Computational Mathematics + Machine Learning + X, where X may be problems in biology, geophysics, soft matter, functional materials, physical chemistry, or fluid and solid mechanics. Their emphasis is on Scientific Machine Learning and Physics-Informed Neural Networks (PINNs) that the CRUNCH group pioneered. 
+tags:
+  - active
+  - applied-math
```
Comment (mirestrepo) [inline our-work/collaborations/crunch.yml:6]: I'm also wondering if we add a tag for `system-administration` or some other tag that relates/identifies to the kind of work you are doing for the crunch group.

**Review by mirestrepo — COMMENTED** [review]
I just left few suggestion, but this looks great!

**Review by mirestrepo — APPROVED** [review]
You can merge this! Thanks again! 🚀

**Pull request merged**